src/main/java/Elon/Parser.java

```diff
@@ -0,0 +1,93 @@
+package Elon;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parser {
+    private Ui ui;
+    private TaskList list;
+    private Scanner scanner;
+    private String[] inputArr;
+    private Storage storage;
+
+    public Parser(Ui ui, TaskList list, Scanner scanner, String[] inputArr, Storage storage) {
+        this.ui = ui;
+        this.list = list;
+        this.scanner = scanner;
+        this.inputArr = inputArr;
+        this.storage = storage;
+    }
+    public String[] nextInput(Scanner scanner) {
+        return scanner.nextLine().split(" ");
+    }
+    public void Parse() throws ElonException{
```

Comment:
Since this is a method, please use camelCase for method names, i.e. `parse` instead of `Parse`.